ludwig/trainers/trainer.py

```diff
@@ -181,19 +163,26 @@ def __init__(
         if self.device is None:
             self.device = get_torch_device()
 
+        base_learning_rate = config.learning_rate
+        if self.horovod:
```

Comment:
This will become self.distributed after  https://github.com/ludwig-ai/ludwig/pull/2890/files lands, right?

Reply:
Yes.